src/main/java/org/apache/hadoop/fs/PrestoFileSystemCache.java

```diff
@@ -27,6 +27,8 @@
 public final class PrestoFileSystemCache
         extends FileSystem.Cache
 {
+    private static final int CACHE_LEAK_PROTECTION_THRESHOLD = Integer.parseInt(System.getProperty("hadoopFileSystemCacheLeakProtectionThreshold", "100000"));
```

Comment:
I agree that the `100000` is to large for the default limit. But we need a way to be able to easily increase that. `1000` is the limit that might be reached in real world. Like, say, when someone have Presto installation that is being used by 1000 users? So, i would like to keep this as a JVM property. Because changing it via hadoop configuration is not that trivial, and would require to expose it explicitly via the configuration object in Presto. What do you think?